src/metadata/persistence/favorites.clj

```diff
@@ -14,10 +19,11 @@
      It returns true if the give target has been marked as a favorite, otherwise it returns false.
      It also returns false if the user or target doesn't exist."
   [user ^UUID target-id]
-  (-> (select :favorites
-        (aggregate (count :*) :cnt)
-        (where {:target_id target-id :owner_id user}))
-    first :cnt pos?))
+  (let [q (-> (h/select [[:> :%count.* 0] :is_favorite])
```

Comment:
This is a bit of a change. I made it do `SELECT count(*) > 0 AS is_favorite` rather than doing the `pos?` in clojureland.